_events/2022/2022-02-community-call.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Feb 2022 Community Call
+expires: 2022-02-11
+event_date: "Feb 10, 2022"
+layout: event
+duration: 60
+repeated: false
+category: community-call
+time:
+    - - start: 2022-02-10T17:00:00Z
```

Comment:
Is this formatting okay?

Reply:
Yep - it's a list of lists, sort of. Check out the preview you'll see that the correct date / time is shown! It's kind of fugly and I think it's something @exoticDFT wanted to work on (and I offer to help with that).